tools/test-js-with-node

```diff
@@ -67,6 +67,7 @@ EXEMPT_FILES = make_set(
         "web/src/browser_history.ts",
         "web/src/buddy_list.ts",
         "web/src/click_handlers.js",
+        "web/src/clipboard_handler.ts",
```

Comment:
We should add some tests for the logic rather than exempting it altogether. You can refer to the docs for writing node tests.